projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -90,7 +90,15 @@
 	font-weight: 700;
 }
 
-.contact-form :where(label.consent) {
+:where( .grunion-checkbox-multiple-options legend ),
+:where( .grunion-radio-options legend ) {
+	margin-bottom: 0.25em;
+	padding: 0;
+	font-weight: 700;
+}
+
+
+:where(.contact-form label.consent) {
```

Comment:
I think this specificity reduction is causing `font-weight: 700` to win out, along with allowing themes to accidentally cause the text to be larger than intended. In testing with TwentyTwenty this is what I'm seeing:

| Trunk | This PR |
| --- | --- |
| <img width="687" alt="image" src="https://github.com/user-attachments/assets/6950fcdc-0a58-4a6a-a208-04570f68f836" /> | <img width="656" alt="image" src="https://github.com/user-attachments/assets/566af9ce-ee74-4b9a-bc33-e321795071ae" /> |

<img width="579" alt="image" src="https://github.com/user-attachments/assets/9e77ba16-374e-4536-b7c8-8fafa288c98a" />

Reply:
Oh dang. I'm not sure what the right thing to do is here. 

How much should we pay attention to the styles of every theme? I'm not sure it's a winning game.

Reply:
Maybe for the consent field it's okay to raise specificity. I think the only major result is that global styles changes to label typography will be overwritten. 🤔

Reply:
Thank you for testing that!

Reply:
By Jove, you're right!

Global styles looking good. Here's me updating the option block font appearance:

| Editor | Frontend |
|--------|--------|
| <img width="400" alt="Screenshot 2025-05-22 at 3 43 57 pm" src="https://github.com/user-attachments/assets/d19c8035-8001-41a4-ad01-153a852c07c2" /> | <img width="400" alt="Screenshot 2025-05-22 at 3 44 00 pm" src="https://github.com/user-attachments/assets/54984967-1152-4c32-96ec-bde28c1852a6" /> |

Both are `400` otherwise 👍🏻

Reply:
Done in [c875ed7](https://github.com/Automattic/jetpack/pull/43545/commits/c875ed7a4978b00910e554b87c7c5a990d829df3)